{"name": "Успешное изменение данных пользователя c авторизацией", "status": "failed", "statusDetails": {"message": "AssertionError: assert (403 == 200)\n +  where 403 = <Response [403]>.status_code", "trace": "self = <test_changing_user_data.TestChangingUserData object at 0x00000194A3DA6950>\nregistration_user = [{'email': 'richardsonbrooke@yandex.ru', 'name': 'epeters', 'password': '(0kLq'}, {'email': 'richardsonbrooke@yandex.r...TcyOWVkMjgwMDAxYjRmNjliYyIsImlhdCI6MTczMDAxNTYwMiwiZXhwIjoxNzMwMDE2ODAyfQ.FAEZRmL2lmWbcMz6PbhJg1eXESLrpi74xCAOWrFLWt4']\n\n    @allure.title('Успешное изменение данных пользователя c авторизацией')\n    def test_change_data_auth_user(self, registration_user):\n        ApiStellarBurgers.auth_user(registration_user[1])\n        response = ApiStellarBurgers.change_user(registration_user[2], Data.DATA_CHANGE)\n>       assert response.status_code == 200 and response_message.TRUE in response.text\nE       assert (403 == 200)\nE        +  where 403 = <Response [403]>.status_code\n\ntests\\test_changing_user_data.py:14: AssertionError"}, "start": 1730015602445, "stop": 1730015603514, "uuid": "11d17b40-bf2c-46df-883e-89c50655dc34", "historyId": "c20470ba794f6dc2a6f97fc1abbcc371", "testCaseId": "c20470ba794f6dc2a6f97fc1abbcc371", "fullName": "tests.test_changing_user_data.TestChangingUserData#test_change_data_auth_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_changing_user_data"}, {"name": "subSuite", "value": "TestChangingUserData"}, {"name": "host", "value": "MSI"}, {"name": "thread", "value": "668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_changing_user_data"}]}